treeherder/perf/models.py

```diff
@@ -86,3 +86,64 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return "{} {}".format(self.value, self.push_timestamp)
+
+
+@python_2_unicode_compatible
+class PerformanceAlert(models.Model):
+
+    id = models.AutoField(primary_key=True)
+    repository = models.ForeignKey(Repository)
+    prev_result_set_id = models.PositiveIntegerField()
+    result_set_id = models.PositiveIntegerField()
+    series_signature = models.ForeignKey(PerformanceSignature)
+    is_regression = models.BooleanField()
+    amount_pct = models.FloatField()
+    amount_abs = models.FloatField()
+    prev_value = models.FloatField()
+    new_value = models.FloatField()
+    t_value = models.FloatField()
```

Comment:
eventually we will want to associate these with bug numbers or other comments, not sure if we should add that in now, or in the next round.  Also some type of status field would be useful.